tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -1888,15 +1889,19 @@ telemetry/test_events.py:
 
 telemetry/test_telemetry.py:
   skip:
-    reason: "Skip telemetry test for 201911 and older branches"
+    conditions_logical_operator: or
+    reason: "Skip telemetry test for 201911 and older branches / Unsupported in MGFX topos"
     conditions:
       - "(is_multi_asic==True) and (release in ['201811', '201911'])"
+      - "topo_type in ['m0', 'mx']"
```

Comment:
Do you aim at skipping entire telemetry test on M0/Mx?

Reply:
No. Updated